chapter-1-basics.md

```diff
@@ -44,7 +44,7 @@ only the `Assembling` step should be covered by MORYX. These information result
 in the following command:
 
 ```
-$ moryx new PencilFactory --steps Assembling --products GraphitePencil
+$ moryx new PencilFactory --steps Assembling --products GraphitePencil --template-url https://git-go.europe.phoenixcontact.com/moryx/moryx-template.git --branch moryx
```

Comment:
While you removed some inaccessible links below, you added this one. Apart from that, those parameters shouldn't be necessary to add.